vite.config.js

```diff
@@ -7,7 +7,6 @@ const config = {
   resolve: {
     alias: {
       $docs: resolve("./src/docs"),
-      $lib: resolve("./src/lib"),
```

Comment:
why is one removed and the other stays?

Reply:
before last version of SvelteKit/Vite we had to overwrite the default `$lib` to be able to declare `$docs` as well but, after the migration, it's the contrary, `$lib` should not be re-declared as available in any case. I guess the custom config used to overwrite the default config while now they merge it.

Reply:
ok, thanks for the explanation!